### ISSUE TITLE
Unable to reproduce results

### ISSUE THREAD

**Issue by MarcVisions** [NVlabs/FreeSOLO#8]
Hi,

Thank you for your work. I encountered several issues when using the code.

- I tried to reproduce the results of Freemask by running the first step of the algorithm on the COCO dataset.
However, I'm unable to get the same results as in the provided json: for example, the embedding vector is not included in the annotations of the provided json.  Am I missing something here?

- I ran the code with the provided json on the train2017+unlabeled2017 split, but only get 0.1% mask AP after the first step of FreeSOLO. In particular, ran the `tools/eval_cocoapi.py` script for the class-agnostic evaluation. I noticed that only the pairwise loss is able to go down.

- Evaluating the provided final model (in this repo) does not produce 12.2% AP50 for detection (as claimed in the paper) but only obtains 9.6%. Do I need to post-process the results before evaluating?

Were people able to reproduce this? Thanks.

**Comment by WXinlong** [comment]
Hi MarcVisions,
- The provided json file doesn't include the semantic embeddings. The embeddings can be generated using the provided `demo/inference_freemask.py`.
- Could you please try again and provide the training scripts and logs? The loss_ins terms should go down.
- Did you get the correct segmentation results? The repo is a reimplementation that may affect the box numbers.
